scripts/prepare-for-platform.sh

```diff
@@ -22,6 +22,13 @@ if [ $# -eq 0 ]
     PLATFORM_FOLDER="${PLATFORM}_files"
 fi
 
+if [[ $PLATFORM == 'mobile' ]]; then
+  _localPropertiesPath=./android/local.properties
```

Comment:
These are set in Jenkins via env variables:
https://github.com/status-im/status-react/blob/develop/ci/Jenkinsfile.android#L25
```
    ANDROID_HOME = '/usr/lib/android-sdk'
    ANDROID_SDK_ROOT = '/usr/lib/android-sdk'
    ANDROID_NDK = '/usr/lib/android-ndk'
    ANDROID_NDK_HOME = '/usr/lib/android-ndk'
```
That should work instead of the `local.properties` file right?